runtime/j9vm/jvm.c

```diff
@@ -1567,6 +1572,31 @@ waitForJITServerTermination(JITServer *jitServer)
 	}
 	return rc;
 }
+
+/**
+ * Stops the JITServer.
+ *
+ * @param jitServer double pointer to the JITServer interface
+ *
+ * @returns JITSERVER_OK on success, else negative error code
+ *
+ * @note on return *jitServer is set to NULL
+ */
+static int32_t
+destroyJITServer(JITServer **jitServer)
+{
+	JavaVM *vm = (*jitServer)->jvm;
+	jint rc = (*vm)->DestroyJavaVM(vm);
+	free(*jitServer);
+	*jitServer = NULL;
+	if (JNI_OK == rc) {
+		rc = JITSERVER_OK;
+	} else {
+		rc = JITSERVER_DESTROY_ERROR;
```

Comment:
This loses the reason for why `DestroyJavaVM` failed.  Unfortunate given this is returning JITSERVER error codes.  Tracepoints would help here - something to consider for the future - but not for this PR.